README.md

```diff
@@ -1 +1,196 @@
-# todo-list
\ No newline at end of file
+
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h1><b>To Do list app</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 To Do List App<a name="about-project"></a>
+
+> it a basic To Do list app,and learning more about javascript using webpack
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML-5
+- CSS3
+- Java Script ES 6
+- Linters
+- webpack
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://github.com/techmoves/my-portfolio/blob/htmlcss1/.github/workflows/linters.yml">Linters</a></li>
```

Comment:
It would be nice if your linters link will redirect to the following link to make your readme descriptive and professional :+1:

```sh
https://github.com/microverseinc/linters-config/tree/master/html-css-js
```

Reply:
fixed